src/icon.rs

```diff
@@ -253,26 +302,31 @@ impl Icons {
         m.insert("m3u", "\u{f910}"); // "蘿"
         m.insert("m3u8", "\u{f910}"); // "蘿"
         m.insert("m4a", "\u{f001}"); // ""
+        m.insert("m4v", "\u{f008}"); // ""
         m.insert("magnet", "\u{f076}"); // ""
-        m.insert("markdown", "\u{f48a}"); // ""
-        m.insert("md", "\u{f48a}"); // ""
+        m.insert("markdown", "\u{e609}"); // ""
```

Comment:
the origin Markdown icon seems more popular, maybe we should keep the origin one

Reply:
Sounds good, it has been reverted. I personally like just the arrow